### DISCUSSION TITLE
Langchain does not work

### DISCUSSION THREAD

**Discussion by pvvdevops27** [AntlerVC/firetable#1488] [Support / Q&A] [answered]
I follow the step to create a basic langchain table using the template, de youtube videotutorial and the blog and it doesn't work

I can connect to firebase using the secret, browsless and openai secret but it does not work, when I add an URL the table remains empty.

**Comment by Chadha93** [comment]
Hi @pvvdevops27 Could you please provide any browser error logs, or screenshots, after you add the URL?

**Reply by pvvdevops27** [reply]
Hello I have this error: InsufficientQuotaError, but Im not sure how can I solve it? It is from browserless, firebase , rowy or openai?

**Reply by pvvdevops27** [reply]
message:"invalid json response body at https://chrome.browserless.io/scrape?token=b4587463-5480-4a48-9ebb-16e39502999f reason: Unexpected token e in JSON at position 1"
type:"invalid-json"
}

**Reply by pvvdevops27** [reply]
After I upgrade my openai account I have this error about a simple https://twitch.com website

**Reply by Chadha93** [reply, marked as answer]
Okay, from the error - it is an issue with the browserless token and the invalid body. You'll have to confirm it from their documentation. Due to this it not read any data, when you add a URL and table remains empty in Rowy.
